frontend/src/utils/templates.ts

```diff
@@ -59,13 +59,64 @@ const replacePlaceholders = (baseText: string, replacementConfig: { [key: string
   return newText;
 };
 
-export const confirmationTemplate = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+export const confirmationTemplateSubmission = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+  const PCNS_URL = 'https://pcns-prod-master.apps.silver.devops.gov.bc.ca';
```

Comment:
Can you ref the url from the route instead of hard coding in the production url?